### PR TITLE
WP9757

### PR DESCRIPTION
**Pull request by naglis**

### CODE REVIEW

File: base_tier_validation/models/tier_validation.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                    for td in tier_definitions:
+                        if self.evaluate_tier(td):
+                            sequence += 1
+                            created_trs += tr_obj.create({
+                                'model': self._name,
+                                'res_id': rec.id,
+                                'definition_id': td.id,
+                                'sequence': sequence,
+                            })
+                    # TODO: notify? post some msg in chatter?
```
Comment (paulius-sladkevicius) [inline base_tier_validation/models/tier_validation.py:194]: @naglis FYI we'll need notifications, so question is if we shouldn't add here a func. that we can hook later :-)

File: base_tier_validation/README.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
+
+Contributors
+------------
+
+* Lois Rilo <lois.rilo@eficent.com>
```
Comment (paulius-sladkevicius) [inline base_tier_validation/README.rst:78]: @naglis where is your name?

File: base_tier_validation/README.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+   functionality.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/250/11.0
+
+Known issues / Roadmap
+======================
+
+* In odoo v11 it would be interesting to try to take advantage of ``mail.activity.mixin``.
```
Comment (paulius-sladkevicius) [inline base_tier_validation/README.rst:55]: v12

**Review by paulius-sladkevicius — APPROVED** [review]
If test not fail, looks good :-)

File: base_tier_validation/models/tier_validation.py (hunk 1 above)
Comment (naglis) [reply base_tier_validation/models/tier_validation.py:194]: I don't think that it should be a part of module migration. Please note that  the `12.0-mig-base-tier-validation` branch is the same in the OCA PR.

Comment (paulius-sladkevicius) [reply base_tier_validation/models/tier_validation.py:194]: I agree, it's only note for a future work.

**Review by kkarolis — APPROVED** [review]

**Pull request merged**